filebeat/tests/system/test_modules.py

```diff
@@ -10,6 +10,8 @@
 import logging
 from parameterized import parameterized
 
+EXTENDED_COMPARE = ["elasticsearch"]
```

Comment:
I would hope that pretty soon this setting becomes obsolete because the extended check is the default. I just didn't want to update all modules as part of this PR.

So in general I agree it could be solved differently but I don't think it's worth the effort at the moment.